### PR TITLE
[improve][doc] Update metadataStoreConfigPath description

### PR DESCRIPTION
**Pull request by nodece**
Signed-off-by: Zixuan Liu <nodeces@gmail.com>

### Motivation

Update metadataStoreConfigPath description, link to https://pulsar.apache.org/docs/next/administration-metadata-store/.

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

**Comment by nodece** [comment]
@momo-jun could you review this PR?

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ Pulsar brokers are responsible for handling incoming messages from producers, di
 |statsUpdateFrequencyInSecs||60|
 |statsUpdateInitialDelayInSecs||60|
 |metadataStoreUrl| Metadata store quorum connection string  ||
+| metadataStoreConfigPath | The configuration file path of the local metadata store, see [here](administration-metadata-store.md) for details. |N/A|
```
Comment (momo-jun) [inline site2/docs/reference-configuration.md:145]:
```suggestion
| metadataStoreConfigPath | The configuration file path of the local metadata store. See [Configure metadata store](administration-metadata-store.md) for details. |N/A|
```
Use alternative text to "here" for better accessibility. Other occurrences need an update as well.

**Review by momo-jun — COMMENTED** [review]
@nodece thanks for completing the required doc for #15270.
I have only a minor suggestion.

**Comment by nodece** [comment]
@momo-jun Thanks for your review! Your request has been fixed.

**Review by momo-jun — APPROVED** [review]
LGTM.

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**